### DISCUSSION TITLE
menu are double when any error happen means if field have null

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#12107] [Q&A]
I read the logs when any content fields of content type are null so the page is crashing on live server but on location it showing error like that:

![image](https://user-images.githubusercontent.com/5202291/181375996-71950adc-52a3-4682-a2fd-3910141eed1f.png)

**Comment by hishamco** [comment]
Please write down a reproduce steps using one of the built-in Orchard Core themes

**Reply by saleemhaq** [reply]
> Please write down a reproduce steps using one of the built-in Orchard Core themes

My concern is that: on local I have any exception it showing on page like below

![image](https://user-images.githubusercontent.com/5202291/181378699-65d6d300-42f2-4a98-b62a-dedb76d7611e.png)

but on hosted server it showing double of menu instead of showing error if error any so i read out from logs.

**Reply by hishamco** [reply]
I'm not sure but may be something related to your theme, that's why I told you if you provide a steps to produce it will be good
